tools/Tools.Common/SerializedCmdlets/Microsoft.Azure.Commands.Profile.dll.json

```diff
@@ -330,7 +330,7 @@
           "Type": {
             "Namespace": "Microsoft.Azure.Commands.Profile.Common",
```

Comment:
Whats the use of this file?

Reply:
this is the record of the public interface, used by the breaking change tool